feature-toggle/src/app/feature-card/feature-card.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="mx-5 mt-5 mb-4 ">
+    <div class="row">
+        <div *ngFor="let feature of paginatedFeatures" class="col-md-3 mb-3">
+            <div class="card" [ngClass]="{
+            'status-border-feature': feature.type === 'feature',
+            'status-border-active': feature.type === 'release' && feature.status === 'enabled',
+            'status-border-inactive': feature.type === 'release' && feature.status === 'disabled'
+          }">
+                <div class="card-body">
+                    <h4 class="card-title">{{ feature.name }}</h4>
+                    <p class="card-text">{{ feature.type === 'feature' ? 'Feature Toggle' : 'Release Toggle' }}</p>
+
+                    <!-- Feature Toggle Buttons -->
+                    <div *ngIf="feature.type === 'feature'">
+                        <button type="button" class="btn btn-primary" (click)="openDrawer()">Enable</button>
+                        <button type="button" class="btn btn-primary ms-2" (click)="closeDrawer()">Disable</button>
+
+                    </div>
+
+                    <!-- Release Toggle Button -->
+                    <div *ngIf="feature.type === 'release'">
+                        <button type="button"
+                            [ngClass]="feature.status === 'enabled' ? 'btn btn-danger' : 'btn btn-success'"
+                            (click)="feature.status = feature.status === 'enabled' ? 'disabled' : 'enabled'">
+                            {{ feature.status === 'enabled' ? 'Disable' : 'Enable' }}
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
+
+    <!-- Drawer Component -->
+    <div *ngIf="isOpen"><app-drawer></app-drawer></div>
+</div>
+
+
+<nav aria-label="Page navigation example">
+    <ul class="pagination justify-content-center pb-2">
+        <!-- Previous Button -->
+        <li class="page-item" [class.disabled]="currentPage === 1">
+            <a class="page-link" href="#" (click)="previousPage();$event.preventDefault()" aria-label="Previous">
+                <span aria-hidden="true">&laquo;</span>
+            </a>
+        </li>
+
+        <!-- Page Numbers -->
+        <li class="page-item" *ngFor="let page of [].constructor(totalPages()); let i = index"
```

Comment:
[].constructor(totalPages())
purpose of this?

Reply:
> [].constructor(totalPages()) purpose of this?

This code is from bootstrap documentation ,  [].constructor(totalPages()) creates an array with length equal to number returned by totalPages function, all elements in the array will be undefined initially